cmd/flow-dps-indexer/main.go

```diff
@@ -215,9 +221,12 @@ func run() int {
 		mapper.WithIndexCommit(flagIndexAll || flagIndexCommit),
 		mapper.WithIndexHeader(flagIndexAll || flagIndexHeader),
 		mapper.WithIndexCollections(flagIndexAll || flagIndexCollections),
+		mapper.WithIndexGuarantees(flagIndexAll || flagIndexGuarantees),
```

Comment:
Same question here. Is there any value in separating collection & guarantee indexing?

Reply:
I applied the same logic as for the previous case

Reply:
Yes, I think it's actually right.